api/v1/qdrantcluster_types.go

```diff
@@ -15,8 +15,16 @@ const (
 )
 
 const (
-	// The Annotation key to trigger a restart.
+	// The annotation key to trigger a restart.
+	// The value should be a [RFC3339 formatted] date.
+	// If the value is updated it will  retrigger the restart.
+	// For historical reasons the key doesn't start with `operator.qdrant.com/`
+	// The annotation should be placed on the QdrantCluster instance.
 	RestartedAtAnnotationKey = "restartedAt"
+	// The annotation key to recreate a certain node.
```

Comment:
Since this is go code, could you please rewrite the comment to match [the godoc convention](https://go.dev/blog/godoc)? Something allong these lines:

```
RecreateNodeAnnotationKey is the annotation key that, when set on QdrantCluster instance, recreates
a given node identified by its index.
```

Please apply the same convention on `RestartedAtAnnotationKey`.

Since this is a part of the API, could you please specify if the node index starts at 1 or 0, and what happens if the set value is greater than the number of nodes in the cluster.

Reply:
adressed